openid-4-verifiable-presentations-1_0.md

```diff
@@ -705,6 +715,89 @@ The rules for selecting claims to send are defined in (#selecting_claims).
 
 Note that multiple Credential Queries in a request MAY request a presentation of the same Credential.
 
+### Trusted Authorities Query {#dcql_trusted_authorities}
+
+A Trusted Authorities Query is an object representing information that helps to identify an issuer
+or the trust framework that an issuer is belonging to. A Credential is identified as a match
```

Comment:
nit:belongs to

Reply:
thanks for catching that - fixed with last commit